docs/meshcloud.policies.md

```diff
@@ -26,7 +28,8 @@ You may encounter policies in the meshPanel when doing any of the following acti
 
 1. Creating a new project
     1. When adding a meshPlatform with a meshLandingZone to a new project, all policies are evaluated between a 'meshProject' and a 'meshLandingZone'
```

Comment:
```suggestion
    1. When adding a meshPlatform with a meshLandingZone to a new project, all policies are evaluated between a 'meshProject' and a 'meshLandingZone'
```
Can we make this a bullet list instead of a numbered list. I dont think the order matters

Reply:
Sure